packages/desktop-client/src/components/modals/ConfirmUnlinkAccount.tsx

```diff
@@ -0,0 +1,57 @@
+// @ts-strict-ignore
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏new files should follow TS strict-mode.

```suggestion
```

Reply:
I copied/pasted an existing modal. ;-)

Fixed.